### PR TITLE
Enhance API tester

### PR DESCRIPTION
**Pull request by yurabakhtin**
**What kind of change does this PR introduce?** (check at least one)

- [x] Feature

**The PR fulfills these requirements:**

**Other information:**
Additional enhancements for API tester related to https://github.com/humhub/humhub/pull/4960

**NOTE:** I think we should wait when all external module API endpoints will be moved to its modules from RESTful API module, because probably the `ApiTester` will be changed.

### CODE REVIEW

File: protected/humhub/tests/codeception/_support/ApiTester.php [hunk 1]
```diff
@@ -25,6 +26,25 @@ class ApiTester extends \Codeception\Actor
 {
     use _generated\ApiTesterActions;
 
+    public $spaces = [
```
Comment (luke-) [inline protected/humhub/tests/codeception/_support/ApiTester.php:29]: Is there no other way to access fixture data? Isn't that kind of duplicated?

Comment (yurabakhtin) [reply protected/humhub/tests/codeception/_support/ApiTester.php]: @luke I have implemented a solution in commit https://github.com/humhub/humhub/commit/456f61b3eab8a8390921635b2054725c9569a695. If it is ok we can factorize this with https://github.com/humhub/humhub/blob/enh/4960-api-tester/protected/humhub/tests/codeception/_support/AcceptanceTester.php#L64-L69 and https://github.com/humhub/humhub/blob/enh/4960-api-tester/protected/humhub/tests/codeception/_support/FunctionalTester.php#L238-L243

**Review by luke- — APPROVED** [review]
@yurabakhtin Thanks!

File: protected/humhub/tests/codeception/_support/ApiTester.php (hunk 1 above)
Comment (luke-) [reply protected/humhub/tests/codeception/_support/ApiTester.php]: @yurabakhtin Thanks, I didn't know it was like that in other places too. Gladly also implement there also there.

Comment (yurabakhtin) [reply protected/humhub/tests/codeception/_support/ApiTester.php]: @luke- Should I factorize this with 2 other places?

Comment (luke-) [reply protected/humhub/tests/codeception/_support/ApiTester.php]: @yurabakhtin Yes

Comment (yurabakhtin) [reply protected/humhub/tests/codeception/_support/ApiTester.php]: @luke- Ok, commit https://github.com/humhub/humhub/pull/4976/commits/11f7cbaf37b935ec30550ce2d433fe96af71ff7a

**Comment by luke-** [comment]
@yurabakhtin Can you please check the tests: https://github.com/humhub/humhub/pull/4976/checks?check_run_id=2238573017

**Comment by yurabakhtin** [comment]
@luke- 

> Can you please check the tests: https://github.com/humhub/humhub/pull/4976/checks?check_run_id=2238573017

Fixed in commit https://github.com/humhub/humhub/pull/4976/commits/4592417f08e4541a45d546ea6c87b12740f75749, the errors were because acceptance testers have no the methods to get fixtures, so I decided to get the date from DB.

**Review by luke- — APPROVED** [review]

**Review by luke- — APPROVED** [review]

**Pull request merged**